src/components/ConversationSettings/DangerZone.vue

```diff
@@ -30,31 +30,31 @@
 				<p class="danger-zone__hint">
 					{{ t('spreed', 'Once a conversation is left, to rejoin a closed conversation, an invite is needed. An open conversation can be rejoined at any time.') }}
 				</p>
-				<button @click.prevent.exact="leaveConversation">
+				<Button type="secondary" @click.prevent.exact="leaveConversation">
```

Comment:
Secondary can not be used.
Primary? Tertiary? Warning?

cc @marcoambrosini

Reply:
Shouldn't we just not change the secondaries for now?

Reply:
Also fine.

But in this case since it's danger zone anyway the yellow/orange warning would also work in my opinion

Reply:
lemme try with tertiary

Reply:
Warning looking good for now!

<img width="949" alt="Screenshot 2022-03-16 at 15 15 18" src="https://user-images.githubusercontent.com/26852655/158610036-a159c548-35d7-4773-9ea1-6ab88671a51a.png">